README.md

```diff
@@ -87,7 +88,7 @@ Runs:
 * __js(gulp, config)__ Config accepts:
 	- js: `String` Path to your main JavaScript file. (Default: './main.js' and checks your bower.json to see if it's in its main key)
 	- buildJs: `String` Name of the built JavaScript bundle. (Default: 'main.js')
-	- buildFolder: `String` Path to directory where the built file will be created. (Default: './build/')
+	- buildFolder: `String` Path to directory where the built file will be created. If set to `null`, files won't be saved. (Default: './build/')
```

Comment:
I had a similar issue in [`o-errors`](https://github.com/Financial-Times/o-errors/blob/master/test/test_errors.js#L81). I originally had the config key named `disabled` so setting it to a truthy value would disable the module, and any falsey value, including undefined, would leave the module enabled.

This meant, by default, if you omitted `disabled` from the config you'd get the default 'enabled' behaviour.

Andrew suggested the config be named 'enabled', which I kind of agreed with.  Except I wanted it to be enabled by default if it was undefined.  [The code for this logic is here](https://github.com/Financial-Times/o-errors/blob/master/src/js/oErrors.js#L77).

I'd really avoid using null to have any semantic meaning.